web/src/components/network/IpSettingsForm.tsx

```diff
@@ -143,10 +138,14 @@ export default function IpSettingsForm() {
     );
   };
 
+  console.log("Method", method);
+  console.log("ConnectionMethod.AUTO", ConnectionMethod.AUTO);
+  console.log("ConnectionMethod.MANUAL", ConnectionMethod.MANUAL);
```

Comment:
this looks like debug lines that should not be merged

Reply:
Right!